go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/webhook
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.5
```

Comment:
May be unimportant, but the latest 1.22 is `go1.22.6`.

Reply:
My goal is to match rancher/rancher, which is tracking `go01.22.5` on its release v2.8 branch https://github.com/rancher/rancher/blob/release/v2.8/go.mod

Reply:
(After the code freeze, perhaps we should bump both. But for release purposes, I think it's mostly  just important to have them in sync.)